### PR TITLE
Revert non-forward-compatible changes for PriorBox & PriorBoxClustered operations

### PR DESCRIPTION
**Pull request by achetver**
### Details:
 - Revert changes for some attributes values in IR from `true`/`false` to `1`/`0` because IE 2021.2 cannot read this values.

### Tickets:
 - 49897

Code:
* [x]  Comments
* [x]  Code style (PEP8)
* [x]  Transformation generates reshape-able IR
* [x]  Transformation preserves original framework node names

Validation:
* [x]  Unit tests: N/A - No new implemented functions;
* [x]  Framework operation tests: N/A - No new enabled operations;
* [x]  Transformation tests: N/A - No new implemented transformations;
* [x]  Model Optimizer IR Reader check: Done manually.

Documentation:
* [x]  Supported frameworks operations list: N/A - No new supported operations;
* [x]  Supported **public** models list: N/A - Nothing to add;
* [x]  User guide update: N/A - Nothing to update.

### CODE REVIEW

**Comment by achetver** [comment]
@pavel-esir @iimironov @evolosen please, review these changes.

File: docs/ops/detection/PriorBoxClustered_1.md [hunk 1]
```diff
@@ -20,10 +20,10 @@
 
   * **Description**: *clip* is a flag that denotes if each value in the output tensor should be clipped within [0,1].
   * **Range of values**:
-    * false - clipping is not performed
-    * true  - each value in the output tensor is within [0,1]
+    * false or 0 - clipping is not performed
+    * true or 1  - each value in the output tensor is within [0,1]
   * **Type**: boolean
-  * **Default value**: true
+  * **Default value**: true or 1
```
Comment (lazarevevgeny) [inline docs/ops/detection/PriorBoxClustered_1.md:26]: Remove this change

File: docs/ops/detection/PriorBox_1.md [hunk 2]
```diff
@@ -28,20 +28,20 @@
 
   * **Description**: *flip* is a flag that denotes that each *aspect_ratio* is duplicated and flipped. For example, *flip* equals 1 and *aspect_ratio* equals to "4.0,2.0" mean that aspect_ratio is equal to "4.0,2.0,0.25,0.5".
   * **Range of values**:
-    * false - each *aspect_ratio* is flipped
-    * true  - each *aspect_ratio* is not flipped
+    * false or 0 - each *aspect_ratio* is flipped
+    * true or 1  - each *aspect_ratio* is not flipped
   * **Type**: boolean
-  * **Default value**: false
+  * **Default value**: false or 0
```
Comment (lazarevevgeny) [inline docs/ops/detection/PriorBox_1.md:34]: remove this change

File: docs/ops/detection/PriorBox_1.md [hunk 3]
```diff
@@ -28,20 +28,20 @@
...
 
   * **Description**: *clip* is a flag that denotes if each value in the output tensor should be clipped to [0,1] interval.
   * **Range of values**:
-    * false - clipping is not performed
-    * true - each value in the output tensor is clipped to [0,1] interval.
+    * false or 0 - clipping is not performed
+    * true or 1 - each value in the output tensor is clipped to [0,1] interval.
   * **Type**: boolean
-  * **Default value**: false
+  * **Default value**: false or 0
```
Comment (lazarevevgeny) [inline docs/ops/detection/PriorBox_1.md:44]: remove this change

**Review by lazarevevgeny — CHANGES_REQUESTED** [review]

File: docs/ops/detection/PriorBoxClustered_1.md (hunk 1 above)
Comment (achetver) [reply docs/ops/detection/PriorBoxClustered_1.md]: Done.

File: docs/ops/detection/PriorBox_1.md (hunk 2 above)
Comment (achetver) [reply docs/ops/detection/PriorBox_1.md]: Done.

File: docs/ops/detection/PriorBox_1.md (hunk 3 above)
Comment (achetver) [reply docs/ops/detection/PriorBox_1.md]: Done.

**Review by evolosen — APPROVED** [review]

**Review by pavel-esir — APPROVED** [review]
Looks good to me. Personally @achetver explained why these changes are needed.

**Comment by achetver** [comment]
@lazarevevgeny This PR is ready for your review.

**Review by lazarevevgeny — APPROVED** [review]

**Pull request merged**